go/vt/vtgate/evalengine/eval_result.go

```diff
@@ -730,6 +730,12 @@ func (er *EvalResult) setBindVar1(typ sqltypes.Type, value []byte, collation col
 		}
 		// TODO: type32
 		er.setInt64(ival)
+	case sqltypes.Uint32:
+		ival, err := strconv.ParseUint(string(value), 10, 32)
+		if err != nil {
+			ival = 0
```

Comment:
You must return an error here just like the other cases.

Reply:
Thanks @vmg! I updated to `throwEvalError(err)` 

Please note that the other int cases are not throwing an error in the current state. For example:
https://github.com/vitessio/vitess/pull/9757/files#diff-8bd3f97165c3b882919fff81a641216b78e2c23ba08ca3d0e2d8dab3574bc72aL735-L737

Maybe it makes sense to change all of them to throw?

Reply:
Huh, I see now that we have inconsistent behavior there. I'll fix it in a separate PR. Thanks for the heads up!